### PR TITLE
Add proof-of-concept telemetry event

### PR DESCRIPTION
**Pull request by inventhouse**
Add proof-of-concept tracking event
Also make it easier to `generate_static_pages` that are _not_ minified for debugging

Please review:
>  M public/static-checkout/generate_static_pages.py
>  M public/static-checkout/generate_static_pages.sh
>  M public/static-checkout/unminified.js

### CODE REVIEW

File: public/static-checkout/generate_static_pages.py [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import csv
 # Combine minified html/css/js into one string
 with open('dist/minified.html', 'r') as file:
-    html = file.read().replace('\n', '')
```
Comment (MatthewPiatetsky) [inline public/static-checkout/generate_static_pages.py:4]: why was this removed?

Comment (inventhouse) [reply public/static-checkout/generate_static_pages.py:4]: They appear to be unnecessary in the minified case (based on my testing) and they interfere with generating non-minified pages for debugging.

File: public/static-checkout/unminified.js [hunk 2]
```diff
@@ -169,6 +200,12 @@ document.addEventListener('DOMContentLoaded', function () {
   renderCountryState();
 });
 
+/* eslint-disable no-unused-vars */
+window.onload = function sendPageLoaded(event) {
+  sendRev1074Event('static.page_loaded', {});
```
Comment (MatthewPiatetsky) [inline public/static-checkout/unminified.js:205]: is this the only event we need?

Comment (inventhouse) [reply public/static-checkout/unminified.js:205]: No, there is at least a couple of others (see REV-1116)

**Pull request merged**